spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -1038,6 +1046,128 @@ public void testInvalidAPIUsage() {
         "Cannot set strategy", () -> actions().rewriteDataFiles(table).sort(SortOrder.unsorted()).binPack());
   }
 
+  @Test
+  public void testRewriteJobOrderBytesAsc() {
+    Table table = createTablePartitioned(4, 2);
```

Comment:
Within these tests are we sure that they are actually producing row groups with different sizes of bytes and files?

Reply:
I believe this would end up just making 2 file groups, one for each partition , and they should be the same size? (at least in number of files?)

Reply:
I have added uneven files under each partition now. I checked using debugger to ensure bytes in size and number of files are different.

Reply:
For `BYTES_ASC`:
<img width="1229" alt="1" src="https://user-images.githubusercontent.com/4561678/162905414-0369f5d0-e795-40cc-9d92-82c34ff7ea7e.png">

Reply:
Sounds good to me, Just to make sure I would just have an assert in the test that sorting in both directions is different :) That way any future changes to parquet protocols would be automatically checked. 

Ie Asstert expected == actual && expected.reversed != actual

Reply:
Makes sense to me. I have pushed the change.